Chosen select for ajaxupload

reilldesign opened the pull request:
Improve user experience for the drag & drop ajaxupload. This pull request adds the jQuery chosen plugin to the select box.

reilldesign committed:
Update and rename default.php to default-chosen.php

reilldesign committed:
Rename default-chosen.php to default.php

Erftralle commented:
Do you think there is a chance to get the "chosen" select box red bodered in case of an validation error

![valid-err-chosen](https://user-images.githubusercontent.com/10573985/48360236-11f11180-e69f-11e8-9253-7ee78aaed9b5.png)

like it is with the standard select box?

![valid-err](https://user-images.githubusercontent.com/10573985/48360304-3947de80-e69f-11e8-82e6-ce6f2c6f3ec0.png)

reilldesign committed:
Update joomcategory.php

Adding the class "chzn-done" to the select box does the job.

Now the style "select.chzn-done.invalid + .chzn-container.chzn-container-single > a.chzn-single" in line 6019 of the Joomla 3 isis template.css file is working

reilldesign commented:
I added a commit for joomcategory.php to solve the problem.

Erftralle commented:
:+1: 
I have tested this item :white_check_mark:  successfully. Thank you.
